rotkehlchen/tests/api/test_data_import.py

```diff
@@ -673,3 +674,21 @@ def test_kucoin_history_import(rotkehlchen_api_server):
         assert assert_proper_sync_response_with_result(response) is True
 
     assert_kucoin_import_results(rotki)
+
+
+def test_blockpit_history_import(rotkehlchen_api_server):
+    """Test that data import works for blockpit csv files"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    dir_path = Path(__file__).resolve().parent.parent
+
+    filepath = dir_path / 'data' / 'blockpit_transactions.csv'
+    json_data = {'source': 'blockpit', 'file': str(filepath)}
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            'dataimportresource',
+        ), json=json_data,
+    )
```

Comment:
```suggestion 
    response = requests.put(
        api_url_for(
            rotkehlchen_api_server,
            'dataimportresource',
        ), json={'source': 'blockpit', 'file': str(filepath)},
    )
```

Reply:
have you checked if the str() is needed?

Reply:
If I remove `str()` it does this:
```
E       TypeError: Object of type PosixPath is not JSON serializable
```